app/views/submissions/_details_form.html.erb

```diff
@@ -0,0 +1,53 @@
+<div data-controller="submission-details">
+    <% rand = SecureRandom.hex(8) %>
+    <%= form_with(model: @submission, url: submission_path(@submission), class: "maxw-mobile-lg") do |form| %>
+        <div class="usa-checkbox" id="eligible-for-evaluation">
+            <input
+            class="usa-checkbox__input usa-checkbox__input--tile"
+            type="checkbox"
+            id="<%= "#{rand}-judging-status-selected" %>"
+            value="selected"
+            <%= "disabled" if @submission.judging_status == "winner" %>
```

Comment:
it looks like #321 will merge first so I wanted to leave a reminder to update this disabled state from that logic once it's merged:
https://github.com/GSA/Challenge_platform/blob/bde972de60cc063c08db949de8f440b6f0d3e2b3/app/models/submission.rb#L78-L84
(also the winner status disabled below)